config.go

```diff
@@ -1826,7 +1830,7 @@ func parseRPCParams(cConfig *lncfg.Chain, nodeConfig interface{},
 	case "bitcoind", "litecoind":
 		nConf := nodeConfig.(*lncfg.Bitcoind)
 		rpcUser, rpcPass, zmqBlockHost, zmqTxHost, err :=
-			extractBitcoindRPCParams(netParams.Params.Name, confFile)
+			extractBitcoindRPCParams(netParams.Params.Name, nConf.Dir, confFile, nConf.RPCCookie)
```

Comment:
pls wrap at 80 chars

Reply:
Wrapped, not sure about Go/LND's wrapping conventions so please double-check.